lighthouse-ci/README.md

```diff
@@ -0,0 +1,119 @@
+# Lighthouse CI Usage
+
+In order to prevent unexpected ad performance regressions, 
+[Lighthouse CI](https://github.com/GoogleChrome/lighthouse-ci) can be integrated 
+with your existing CI to ensure that scores do not regress with code changes.
+
+"Lighthouse CI is a suite of tools that make continuously running, saving, 
+retrieving, and asserting against 
+[Lighthouse](https://github.com/GoogleChrome/lighthouse) results as easy as 
+possible."
+
+Today, this tool is compatible with GitHub Actions, Travis CI, Circle CI, GitLab CI, Jenkins, and Google Cloudbuild.
+
+If not familiar with Lighthouse CI, please familiarize yourself with their 
+[Getting Started documentation](https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/getting-started.md#github-app-method-recommended) 
+prior to starting.
+
+## Config
+
+Lighthouse CI natively supports plugin usage and Publisher Ads Audits can be 
+easily included. Simply include the plugin in the generated reports by setting 
```

Comment:
"generated reports" doesn't seem very clear to me. How about: "To run Publisher Ads Audits in your Lighthouse CI reports, simply set the ..."